pkg/e2e/framework/common.go

```diff
@@ -40,10 +40,10 @@ const (
 )
 
 // RandomString returns a random 6 character string.
-func RandomString() string {
+func RandomString(clusterName string) string {
 	randID := string(uuid.NewUUID())
 
-	return randID[:6]
+	return clusterName + randID[:6]
```

Comment:
can we `fmt.Sprintf("%s-%s",
			c clusterName,
			randID,
		)`
to make it more readable and be consistent with https://github.com/openshift/installer/blob/master/pkg/asset/machines/gcp/machinesets.go#L45